lib/cli.js

```diff
@@ -145,6 +145,9 @@ cli.generateParserString = function generateParserString(opts, grammar) {
     if (opts.moduleName) {
         settings.moduleName = opts.moduleName;
     }
+    if (opts.moduleMain) {
+        settings.moduleMain = opts.moduleMain;
```

Comment:
No parsing necessary here?

Reply:
I think it's just a string of source code, isn't it?